src/test-error/templates.js

```diff
@@ -0,0 +1,256 @@
+import ERROR_CATEGORY from './error-category';
+import TYPE from './type';
+import buildDiff from './diffs';
+
+function getMsgPrefix (err) {
+    return typeof err.message !== 'undefined' && err.message !== null ?
+           `<error-category>${ERROR_CATEGORY.assertError}</error-category>"${err.message}" assertion` :
+           `<error-category>${ERROR_CATEGORY.assertError}</error-category>Assertion`;
+}
+
+function getDiffRelatedStr (err) {
+    if (err.isArrays)
+        return `\nArrays differ at index <js>${err.key}</js>:\n\n`;
+    else if (err.isObjects)
```

Comment:
No `else if` here since we have `return`. Here and in other places, please separate `if`-statements with the blank line.